examples/informers/ServiceCollectionKubernetesExtensions.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Linq;
+using System.Threading;
+using k8s;
+using k8s.Informers;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Rest;
+using Microsoft.Rest.TransientFaultHandling;
+
+namespace informers
+{
+    public static class Extensions
+    {
+        public static IServiceCollection AddKubernetes(this IServiceCollection services)
+        {
+            services.AddHostedService<ControllerService>();
+            var controllers = AppDomain.CurrentDomain.GetAssemblies()
```

Comment:
I would refrain from any scanning like this being built in to core methods like AddKubernetes. 

This should also be split into its own package.

Reply:
This is what I suggested: modularize the SDK into multiple packages. I'm still waiting for a blessing from maintainers to do so before I put shovels in the ground

Reply:
I was trying to create similar pattern to what ASP.NET core uses, which also does assembly scanning to discover Controllers.

Reply:
@macsux Don't 😄

Reply:
Lol. Isn't asp.net your baby? I'm trying to make this as intuitive as possible for dotnet developer to write custom controllers, operators. I hate this status quo atm where kubernetes = write everything in Go. C# is just so elegant

Reply:
Yes but scanning assemblies is my arch-nemesis. MVC is the only thing that does it and I'm actively protesting against it 😄

Reply:
Ok, I've switched it to be explicit in the sample project. Stay tuned for some kickass sample .NET k8s operators /w blog posts built on top of all this work :)